### DISCUSSION TITLE
Arrow icons

### DISCUSSION THREAD

**Discussion by reginastracii** [prateekmedia/netspeedsimplified#19] [Q&A] [answered]
I'm getting squares instead the up and down arrows, sum and the double arrow work fine. 
🡳 and 🡱 don't work for me could be a local issue 

![Screenshot from 2021-04-10 21-24-02](https://user-images.githubusercontent.com/22214293/114283762-55543380-9a43-11eb-9c15-edc109b4a15d.png)

**Comment by prateekmedia** [comment, marked as answer]
Either change fonf or change Icon set from preferences.

**Reply by fmorato** [reply]
What preferences do you mean, Gnome Tweaks?

**Reply by prateekmedia** [reply]
> What preferences do you mean, Gnome Tweaks?

Do you have extensions app for gnome? It comes preinstalled mostly on all distros.

In that extensions app you will find net speed Simplified and on the right side you see settings icon, clicking that will reveal all the good things this extension has as an option.

**Reply by fmorato** [reply]
Thanks for your answer. I already know about the extension preferences (really good options :+1:), but I don't see an option to change fonts or icons there. 
![image](https://user-images.githubusercontent.com/12693347/126360007-cb1ffbc3-1c62-446c-a0f2-f81a8dde07ac.png)

**Reply by prateekmedia** [reply]
You can scroll to reveal all the beauty or just maximise the window.

**Reply by fmorato** [reply]
Oh wow.. '^^ thanks, I hadn't tried to scroll.

**Comment by reginastracii** [comment]
I tried switching through the fonts, with no effect. I changed the  🡳 and 🡱 with arrows which do work for me  in the function DIcons(iNum).  gnome 40, fonts various (cantarell, but tried many others) for reference.

**Comment by idanka** [comment]
the same problem: missing up and down arrows 
Ubuntu 21.04 / Ubuntu regular and bold.

But it doesn't matter which font I choose.

![Képernyőkép – 2021-07-12 09-55-51](https://user-images.githubusercontent.com/3587195/125251216-5e30fb80-e2f7-11eb-9b48-d9322cc08fe1.png)

**Comment by idanka** [comment]
Sorry solved:)

Choose icon sett
